bin/compile

```diff
@@ -4,8 +4,9 @@
 set -eo pipefail
 
 mkdir -p "$1" "$2"
+mkdir -p "$2/go-cache"
 build=$(cd "$1/" && pwd)
-cache=$(cd "$2/" && pwd)
```

Comment:
A check to ensure that `$2/go-cache` exists need to happen before the subshell, otherwise cache is set to ""

```console
$ echo "'$(cd nonexistant && pwd)'"
cd: no such file or directory: nonexistant
''
```